download.rst

```diff
@@ -156,6 +156,6 @@ Other packages that Nengo can interact with include
 - `scikit-learn <http://scikit-learn.org/stable/>`_
 - `pytest <https://docs.pytest.org/en/latest/>`_
 
-Other parts of the :ref:`Nengo ecosystem <Nengo projects>`
+Other parts of the :ref:`Nengo ecosystem <Nengo ecosystem>`
```

Comment:
Because the reference and the text for the reference are the same, I believe you don't need the angle bracketed part here, just `:ref:'Nengo ecosystem'` (but with backticks -- can't put backticks within backticks in a Github comment, you get it)